File: mtothePowern.c
//Program to calculate m to the power n.

#include<stdio.h>
int main()
{
	int base, power, counter;
	long long result = 1;
	printf("Enter a base number: ");
	scanf("%d", &base);
	printf("Enter a power number: ");
	scanf("%d", &power);
	for(counter=1; counter <= power; counter++)
	{
		result *= base;
	}
	printf("%d to the power of %d is %d", base, power, result);
	return 0;
}

File: Programs/mToThePowern.py
#Program to calculate m to the power n.

base = int(input("Enter base number: "))
exponent = int(input("Enter exponent number: "))
result = 1
for exponent in range(0, exponent + 1, 1):
	result = result * base
print(base, "to the power of", exponent, "is", result)